flink-end-to-end-tests/test-scripts/test_mesos_heap_memory.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
The file name does not sounds descriptive.
I would suggest something like `test_mesos_streaming_heap_state.sh`.

Reply:
Sounds good to me.